Sources_v3/StreamChatUI/UIConfig.swift

```diff
@@ -83,3 +86,20 @@ public extension UIConfig {
         }
     }
 }
+
+// MARK: - CurrentUser
+
+public extension UIConfig {
+    struct CurrentUserUI {
+        public var currentUserView: CurrentChatUserAvatarView<ExtraData>.Type
+        public var currentUserAvatarView: AvatarView.Type
```

Comment:
I'm confused by the naming here. What is what?

Reply:
fixed in 979c9e0bc42e75728c9641a26f7c971d766da7bf